spring-cloud-generator/pom.xml

```diff
@@ -110,7 +110,28 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+              <artifactId>maven-assembly-plugin</artifactId>
+              <configuration>
+                <archive>
+                  <manifest>
+                    <mainClass>com.google.cloud.generator.spring.Main</mainClass>
+                  </manifest>
+                </archive>
+                <descriptorRefs>
+                  <descriptorRef>jar-with-dependencies</descriptorRef>
```

Comment:
This is in order to run the spring generator jar using dependencies such as protobuf classes. This was something bazel took care about with the previous setup.